contexts/credentials/v2

```diff
@@ -134,6 +134,20 @@
       }
     },
 
+    "JsonSchema": {
+      "@id": "https://www.w3.org/TR/vc-json-schema#JsonSchema",
```

Comment:
```suggestion
      "@id": "https://www.w3.org/TR/vc-json-schema#JsonSchema",
```

VCDM Vocabulary or a different one? I'm fine either way.

Reply:
update

Reply:
```suggestion
      "@id": "https://w3.org/2018/credentials#JsonSchema",
```

No, still wrong, I believe

Fixed to align with this:

https://github.com/w3c/vc-data-model/pull/1218/files